### PR TITLE
Update wizard naming to App Control Wizard

### PR DESCRIPTION
**Pull request by jgeurten**
Given the recent rebranding of WDAC to Application Control for Business, the Wizard UI has been updated to match the new branding. 

Instances of WDAC have been renamed to Application Control for Business, Application Control, or simply, App Control. 

All UI elements, logs and text have been updated. The package family name and exe/dll binaries still use the WDACWizard naming convention to ensure updates are successful. You can also search Windows Search for WDAC:

image

image

image

### CODE REVIEW

File: WDAC-Policy-Wizard/app/MSIX/AllowMicrosoft - SingleFormat.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <SiPolicy xmlns="urn:schemas-microsoft-com:sipolicy">
-  <VersionEx>10.1.1.0</VersionEx>
+  <VersionEx>10.0.5.0</VersionEx>
```
Comment (bowerscd-corp) [inline WDAC-Policy-Wizard/app/MSIX/AllowMicrosoft - SingleFormat.xml:3]: this doesn't look related

File: WDAC-Policy-Wizard/app/MSIX/AppxManifest.xml [hunk 2]
```diff
@@ -4,13 +4,13 @@
   xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
   xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities">
   <Identity Name="Microsoft.WDAC.WDACWizard"
-            Version="2.4.4.0"
+            Version="2.0.0.0"
```
Comment (bowerscd-corp) [inline WDAC-Policy-Wizard/app/MSIX/AppxManifest.xml:7]: version rollback?

File: WDAC-Policy-Wizard/app/MSIX/CreateScannedPolicy.ps1 [hunk 3]
```diff
@@ -0,0 +1,45 @@
+﻿# ScanPath: path to scan
```
Comment (bowerscd-corp) [inline WDAC-Policy-Wizard/app/MSIX/CreateScannedPolicy.ps1:1]: This doesn't look related to the rebranding

File: WDAC-Policy-Wizard/app/MSIX/DefaultWindows_Audit - SingleFormat.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <SiPolicy xmlns="urn:schemas-microsoft-com:sipolicy">
-  <VersionEx>10.2.0.0</VersionEx>
+  <VersionEx>10.3.0.0</VersionEx>
```
Comment (bowerscd-corp) [inline WDAC-Policy-Wizard/app/MSIX/DefaultWindows_Audit - SingleFormat.xml:3]: another different version?

File: WDAC-Policy-Wizard/app/MSIX/DefaultWindows_Audit - SingleFormat.xml [hunk 5]
```diff
@@ -22,9 +22,6 @@
     <Rule>
       <Option>Enabled:Update Policy No Reboot</Option>
     </Rule>
-    <Rule>
-      <Option>Enabled:Dynamic Code Security</Option>
-    </Rule>
```
Comment (bowerscd-corp) [inline WDAC-Policy-Wizard/app/MSIX/DefaultWindows_Audit - SingleFormat.xml:27]: Seems unrelated?

File: WDAC-Policy-Wizard/app/MSIX/WDAC Wizard.deps.json [hunk 6]
```diff
@@ -15,21 +15,21 @@
           "Microsoft.PowerShell.ConsoleHost": "7.4.2",
           "Microsoft.WSMan.Management": "7.4.2",
           "System.Management.Automation": "7.4.2",
-          "runtimepack.Microsoft.Windows.SDK.NET.Ref": "10.0.22621.31"
+          "runtimepack.Microsoft.Windows.SDK.NET.Ref": "10.0.22621.34"
```
Comment (bowerscd-corp) [inline WDAC-Policy-Wizard/app/MSIX/WDAC Wizard.deps.json:18]: ?

**Review by bowerscd-corp — CHANGES_REQUESTED** [review]

**Review by bowerscd-corp — APPROVED** [review]
runtimepack.Microsoft.Windows.SDK.NET.Ref comment nonwithstanding, LGTM

**Comment by Obhiao1997** [comment]
I have the below version of Windows 
Windows 10 22H2
Feature Pack
1000.19060.1000.0
OS build 19045.5131

and I am unable to install WDAC MSIX installer - the file does not open up with install in the dropdown menu rather it opens up but I do not see the install menu - run as administrator option

**Comment by Obhiao1997** [comment]
please can you make this work

**Pull request merged**